azure-pipelines/dotnet.yml

```diff
@@ -9,6 +9,13 @@ steps:
 - powershell: azure-pipelines/dotnet-test-cloud.ps1 -Configuration $(BuildConfiguration) -Agent $(Agent.JobName) -PublishResults
   displayName: 🧪 dotnet test
   condition: and(succeeded(), ${{ parameters.RunTests }})
+  # condition: and(and(succeeded(), ${{ parameters.RunTests }}), !eq(variables['BuildConfiguration'], 'ReleaseMac'))
+
+- powershell: azure-pipelines/dotnet-test-mac-cloud.ps1 -Configuration $(BuildConfiguration) -Agent $(Agent.JobName) -PublishResults
+  script:
+    dotnet build $(RepoRoot)/test/Xunit.StaFact.Tests.Mac/Xunit.StaFact.Tests.Mac.csproj -t:Test -bl:"$ArtifactStagingFolder/build_logs/test.binlog" --diag "$ArtifactStagingFolder/test_logs/diag.log;TraceLevel=info"
```

Comment:
You're missing a `|` at the end of `script: ` if you want to span to the next line.